lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -224,13 +224,17 @@ class CategoryRenderer {
 
     // Add audits without a group first so they will appear first.
     const notAGroup = 'NotAGroup';
+    const hiddenGroup = 'hidden';
     grouped.set(notAGroup, []);
 
     for (const auditRef of auditRefs) {
       const groupId = auditRef.group || notAGroup;
       const groupAuditRefs = grouped.get(groupId) || [];
-      groupAuditRefs.push(auditRef);
-      grouped.set(groupId, groupAuditRefs);
+      // Do not render audit if it is in the 'hidden' group.
+      if (groupId !== hiddenGroup) {
```

Comment:
an alternative to this is 

```js
if (groupId == hiddenGroup) continue;
```

the primary benefit being less indentation.
both are fine, tbh.